src/dbnode/storage/cleanup.go

```diff
@@ -374,10 +374,10 @@ func (m *cleanupManager) cleanupSnapshotsAndCommitlogs() (finalErr error) {
 					// have no impact on correctness as the snapshot files from previous (successful) snapshot will still be
 					// retained.
 					m.metrics.corruptSnapshotFile.Inc(1)
-					m.opts.InstrumentOptions().Logger().WithFields(
-						xlog.NewField("err", err.Error()),
-						xlog.NewField("files", snapshot.AbsoluteFilepaths),
-					).Errorf(
+					m.opts.InstrumentOptions().Logger().With(
+						zap.Error(err),
```

Comment:
Same here with `zap.Error()` at the end.

Reply:
done